api/config/config_test.json

```diff
@@ -103,7 +103,7 @@
     "enableResearchPurposePrompt": false,
     "enableGenomicExtraction": true,
     "enableFireCloudV2Billing" : false,
-    "enableAccessModuleRewrite" : false,
+    "enableAccessModuleRewrite" : true,
```

Comment:
Since the UI does not actually use this, please remove it from ConfigResponse.  That will also make it easier to remove the FF sooner.